modules/ROOT/pages/shared-resources.adoc

```diff
@@ -136,9 +150,12 @@ After completing these steps, Studio includes the domain in the `pom.xml` for yo
 </dependencies>
 ----
 
-To associate an app with a domain outside Studio, just add the domain dependency to the `pom.xml` file as shown in the previous example.
+=== Associate an App with a Domain Outside Studio
+
```

Comment:
Is there anything useful to use as a text transition here, between the title and the first step? Maybe circumstances in which this approach is necessary or preferred?
GLOBALLY look for ways to avoid going from a title straight to Step 1 or a table or a graphic or similar nonnarrative element.